operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

```diff
@@ -93,10 +93,10 @@ The Release Manager \(Amy Blais\) owns the release announcement, including the f
 ## Language Guidelines
 
 1. Use simple language with no buzzwords
-2. Specify Desktop app or Mobile App if a feature is not dependent on the server release
+2. Specify Desktop app or Mobile app if a feature is not dependent on the server release
```

Comment:
I forget if we decided to use "Desktop App" and "Mobile App" when speaking about the Mattermost apps themselves?

Reply:
I've been using "Desktop app" and "Mobile app." My vote is for "Mattermost Desktop app" and "Mattermost Mobile app" and I have updated this PR: https://github.com/mattermost/mattermost-handbook/pull/362 so we have a starting point.

Reply:
@justinegeffen Added a comment in #362: https://github.com/mattermost/mattermost-handbook/pull/362#pullrequestreview-492809359

Reply:
Leaving this as-is for now as it's consistent within the doc. Based on all the conversations around this we're leaning toward Mobile App, Desktop App, and Web App for now.